cmake/clang-query-wrapper.py.in

```diff
@@ -0,0 +1,68 @@
+import sys,os,argparse
+import subprocess
+#credit: https://stackoverflow.com/questions/377017/test-if-executable-exists-in-python
+invocation = "@CLANGQUERY_EXECUTABLE@ "
```

Comment:
Does this entire script need to be a config file?  Perhaps the config file could just contain the necessary parameters and a call to the python script with those parameters?
(This script alrady has an args parser)

Reply:
This was a great suggestion, a lot of the goofiness of the implementation is gone with that step